packages/Frontend/src/containers/AppContainer.jsx

```diff
@@ -1,25 +1,17 @@
 import React, { Component } from 'react';
-import Amplify from 'aws-amplify';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { BrowserRouter } from 'react-router-dom';
+import { auth } from 'firebase';
 import App from '../components/App';
 import logout from '../actions/logout';
-
-Amplify.configure({
-  Auth: {
-    identityPoolId: process.env.AWS_AUTH_IDENTITYPOOLID,
-    region: process.env.AWS_AUTH_REGION,
-    userPoolId: process.env.AWS_AUTH_USERPOOLID,
-    userPoolWebClientId: process.env.AWS_AUTH_USERPOOLWEBCLIENTID,
-    mandatorySignIn: false,
-  },
-});
+import login from '../actions/login';
 
 class AppContainer extends Component {
   static propTypes = {
     isAuthenticated: PropTypes.bool.isRequired,
     logout: PropTypes.func.isRequired,
+    login: PropTypes.func.isRequired,
```

Comment:
This login function actually comes from redux, at the bottom of this file is how we tell redux to pass this in as a prop to AppContainer. We use this login function to add the user to redux for other parts of the app to use.